client/checkout/woopay/email-input-iframe.js

```diff
@@ -497,32 +484,28 @@ export const handlePlatformCheckoutEmailInput = async (
 
 		timer = setTimeout( () => {
 			if ( validateEmail( email ) ) {
-				platformCheckoutLocateUser( email );
+				woopayLocateUser( email );
 			}
 		}, waitTime );
 	} );
 
 	window.addEventListener( 'message', ( e ) => {
-		if ( ! getConfig( 'platformCheckoutHost' ).startsWith( e.origin ) ) {
+		if ( ! getConfig( 'woopayHost' ).startsWith( e.origin ) ) {
 			return;
 		}
 
 		switch ( e.data.action ) {
-			case 'auto_redirect_to_platform_checkout':
+			case 'auto_redirect_to_woopay':
```

Comment:
This should have the same issue as the other action names coming from WooPay, but I actually didn't encounter any problems. Let's keep both cases for now though just to be sure.